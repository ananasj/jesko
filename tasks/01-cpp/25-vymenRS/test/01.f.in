17 20
82 61 27 95 32 13 1 8 36 3 51 50 29 81 97 70 91 12 10 93
42 40 92 81 50 76 53 62 79 17 82 60 29 8 7 60 73 7 19 8
9 69 9 90 49 5 59 91 17 20 36 58 60 27 38 9 55 42 22 85
58 56 45 38 15 51 49 87 9 68 46 70 88 55 59 37 59 69 79 75
89 14 85 100 93 22 60 47 64 34 31 73 89 27 63 3 29 11 42 38
78 87 7 18 93 17 54 4 37 32 30 77 98 14 28 90 88 88 88 3
73 70 75 61 97 89 15 77 100 56 14 29 95 72 46 39 40 51 42 29
35 72 57 32 85 85 21 72 24 60 74 96 29 1 8 77 41 22 54 40
30 19 69 76 43 66 14 82 17 56 62 51 27 19 34 63 55 6 87 78
65 12 73 93 12 32 70 5 5 75 44 34 93 64 61 35 30 27 69 98
82 30 48 60 100 33 22 54 38 8 31 2 20 55 46 83 38 15 87 43
89 83 28 34 46 89 68 27 15 88 24 48 70 23 7 69 55 80 75 92
40 57 45 11 64 91 45 1 57 84 43 46 66 71 31 63 11 98 42 25
86 17 24 55 92 30 75 46 61 49 90 100 6 86 10 69 76 7 21 33
90 64 30 7 34 60 69 44 9 62 20 94 79 43 100 70 24 27 67 84
75 8 36 32 94 97 52 69 3 73 53 92 36 82 50 21 93 19 16 2
80 35 47 10 29 47 79 52 73 46 87 99 53 22 83 46 19 34 67 73
