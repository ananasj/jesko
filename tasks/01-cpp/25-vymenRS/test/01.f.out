82 42 9 58 89 78 73 35 30 65 82 89 40 86 90 75 80
61 40 69 56 14 87 70 72 19 12 30 83 57 17 64 8 35
27 92 9 45 85 7 75 57 69 73 48 28 45 24 30 36 47
95 81 90 38 100 18 61 32 76 93 60 34 11 55 7 32 10
32 50 49 15 93 93 97 85 43 12 100 46 64 92 34 94 29
13 76 5 51 22 17 89 85 66 32 33 89 91 30 60 97 47
1 53 59 49 60 54 15 21 14 70 22 68 45 75 69 52 79
8 62 91 87 47 4 77 72 82 5 54 27 1 46 44 69 52
36 79 17 9 64 37 100 24 17 5 38 15 57 61 9 3 73
3 17 20 68 34 32 56 60 56 75 8 88 84 49 62 73 46
51 82 36 46 31 30 14 74 62 44 31 24 43 90 20 53 87
50 60 58 70 73 77 29 96 51 34 2 48 46 100 94 92 99
29 29 60 88 89 98 95 29 27 93 20 70 66 6 79 36 53
81 8 27 55 27 14 72 1 19 64 55 23 71 86 43 82 22
97 7 38 59 63 28 46 8 34 61 46 7 31 10 100 50 83
70 60 9 37 3 90 39 77 63 35 83 69 63 69 70 21 46
91 73 55 59 29 88 40 41 55 30 38 55 11 76 24 93 19
12 7 42 69 11 88 51 22 6 27 15 80 98 7 27 19 34
10 19 22 79 42 88 42 54 87 69 87 75 42 21 67 16 67
93 8 85 75 38 3 29 40 78 98 43 92 25 33 84 2 73
